aws/secrets.mk

```diff
@@ -1,36 +1,47 @@
 # Macroses
 ########################################################################################################################
+# SSM Wrapper
+SSM_DOCKER_TAG ?= hazelops/ssm-wrapper:latest
```

Comment:
Let's rename this to be more descriptive. Wouldn't `SSM_WRAPPER_DOCKER_IMAGE` or something similar be more logical? Or am I misunderstanding something?

Reply:
it makes sense

Reply:
done